detector-model/src/main/java/org/lcsim/geometry/compact/converter/HPSTracker2019GeometryDefinition.java

```diff
@@ -556,7 +556,7 @@ public static class ModuleL3Bot extends ModuleL13Bot {
         //2019 MRSolt survey
         //protected final static double L3_new_vertical_shift = 0.7 - 0.012;
         //2021 nominal
-        protected final static double L3_new_vertical_shift = 0.7;
+        protected final static double L3_new_vertical_shift = 0.;
```

Comment:
Any clue why Matt Added this 0.7mm. ?

Reply:
My best guess is this was to account for the shims added wrt the engineering run detector. I don't think it was that thick, will confirm with Tim. Also, this isn't even used if you supply survey data for the modules.